src/server_manager/messages/en.json

```diff
@@ -8,6 +8,7 @@
   "aws-lightsail-firewall-4": "Set 'Application' value to 'All TCP+UDP'.",
   "aws-lightsail-firewall-5": "Click 'Save'.",
   "cancel": "Cancel",
+  "outline-documentation": "Manager Resources",
```

Comment:
Are we going to wait for the translations or just create another PR once the translations are ready?